nbs/03_matcher.ipynb

```diff
@@ -34,25 +34,27 @@
    "cell_type": "markdown",
```

Comment:
<p>There is an overloading of the term "single nodes" we are using it for both single vs collection semantics and for matching attributes on nodes before matching topology. See if you can separate these concepts semantically.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/DeanLight/graph_rewrite/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>